### PR TITLE
feat: adding tensor exponential method

### PR DESCRIPTION
**Pull request by Viiktree**
# PR Description

## Related Issue

Close #26236

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

### Socials
https://www.linkedin.com/in/okeavictory/

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Review by lucasalavapena — CHANGES_REQUESTED** [review]
Thanks for your contribution :smile: .

Could you please edit your initial comment with the issue you are closing ("Close #{ISSUE_NUMBER}") and make sure you are passing all your tests (the tests you introduced are failing)! I would recommend running the tests locally or on codespace and running pytest only for the specific test `test_paddle_tensor_exponential_`. Feel free to ping me when you have fixed all your tests!

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py [hunk 1]
```diff
@@ -1927,6 +1927,46 @@ def test_paddle_tensor_exp_(
     )
 
 
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="exp_",
```
Comment (lucasalavapena) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:1933]: Are you sure you are testing the right function?

**Review by lucasalavapena — COMMENTED** [review]
Could you please put the Issue for Close #numbers not this PR!

**Pull request closed without merging**